VideoLocker/res/layout/row_course_list.xml

```diff
@@ -8,67 +8,186 @@
-                    app:avatarCount="6"
-                    app:avatarMaxSize="50dp"
-                    android:gravity="center|left|start"
+                    android:layout_gravity="center_vertical"
+                    android:contentDescription="@string/new_course_content"
+                    android:paddingLeft="3dp"
+                    android:paddingStart="3dp"
+                    android:paddingRight="0dp"
+                    android:paddingEnd="0dp"
+                    android:text="@string/new_course_content"
+                    android:textAllCaps="true"
+                    android:textSize="10sp"
                     tools:targetApi="17" />
+            </LinearLayout>
 
+            <LinearLayout
+                android:id="@+id/schoolcode_layout"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentLeft="true"
+                android:layout_alignParentStart="true"
+                android:layout_marginLeft="10dp"
+                android:layout_marginRight="10dp"
+                android:orientation="horizontal"
+                android:layout_toLeftOf="@id/starting_from_layout"
+                android:layout_toStartOf="@id/starting_from_layout"
+                android:layout_alignTop="@id/starting_from_layout"
+                android:layout_below="@id/course_name"
+                tools:targetApi="17">
+
+                <org.edx.mobile.view.custom.ETextView
+                    android:id="@+id/org_code_tv"
+                    style="@style/regular_text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_vertical"
+                    android:textColor="@color/org_code_txt"
+                    android:textSize="12sp"
+                    android:singleLine="true"/>
+
+                <TextView
+                    android:id="@+id/or_tv"
+                    style="@style/regular_grey_text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_vertical"
+                    android:text="@string/or_symbol"
+                    android:textSize="12sp" />
+
+                <org.edx.mobile.view.custom.ETextView
+                    android:id="@+id/school_code_tv"
+                    style="@style/regular_text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_vertical"
+                    android:ellipsize="end"
+                    android:textColor="@color/org_code_txt"
+                    android:textSize="12sp"
+                    android:singleLine="true"/>
             </LinearLayout>
 
         </RelativeLayout>
 
+
         <RelativeLayout
```

Comment:
Yeah, i need to cleanup the layout code, for now, I've made it to work with minimal cleanup, so that I can see the results of the transformation and make it perfect.